# Output voltage from amplifiers of strain gauges.
# This msg targets communication using rosserial.
# To lighten process on microprocessor, this msg is intended to contain original A/D converter counts.

int32[] array  # Output voltage from amplifiers of strain gauges in A/D converter counts
